MyQuizAdmin/MyQuizAdmin/MainPage.xaml.cs

```diff
@@ -28,8 +28,8 @@ public sealed partial class MainPage : Page
                 new NavMenuItem()
                 {
                     Symbol = Symbol.Contact,
-                    Label = "Page1",
-                    DestinationPage = typeof(Views.Page1)
+                    Label = "GroupView",
+                    DestinationPage = typeof(Views.GroupPage)
```

Comment:
Be aware that the group view will not be the first view so it would be great if you don't register it as the first.
(Also remember [here](https://github.com/DerTieran/MyQuizAdmin/pull/13/files#diff-52977113767945ddfdf53bcca7dd6717R65))